lib/logstash/filters/metrics.rb

```diff
@@ -193,7 +195,7 @@ def flush(options = {})
     return unless should_flush?
 
     event = LogStash::Event.new
-    event.set("message", Socket.gethostname)
+    event.set("host", @host)
```

Comment:
can we keep the key in the event to `message` to avoid bumping the major version?